gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,4 +1,4 @@
-#Tue Aug 27 14:23:52 MSK 2019
+#Sat Sep 07 13:28:22 MSK 2019
```

Comment:
what for?))

Reply:
I had a problem building the project on Travis, so I tried different ways to solve it, including updating the gradle-wrapper.properties file

Reply:
Did it help?

Reply:
No, the problem was different